snapcraft/cli/_errors.py

```diff
@@ -110,6 +108,57 @@ def exception_handler(exception_type, exception, exception_traceback, *,
     sys.exit(exit_code)
 
 
+def _is_send_to_sentry() -> bool:
+    # If ALWAYS has already been selected from before do not even bother to
+    # prompt again.
+    config_errors = None
+    try:
+        cli_config = _CLIConfig()
+        always_value = cli_config.get_option('Sentry', 'always_send')
+        if always_value == 'true':
+            return True
+    except errors.SnapcraftInvalidCLIConfigError as config_error:
+        raise config_error
+        echo.warning('There was an issue while trying to retrieve '
+                     'configuration data: {!s}'.format(config_error))
+        config_errors = config_error
+
+    # Either ALWAYS has not been selected in a previous run so resort to
```

Comment:
s/Either//

Reply:
I was actually missing the **or** :sunglasses: